org.eclipse.xtext.ide/xtend-gen/org/eclipse/xtext/ide/server/Document.java

```diff
@@ -25,7 +25,25 @@
   
   private final String contents;
   
-  public int getOffSet(final Position position) {
+  private final boolean printSourceOnError;
+  
+  /**
+   * @since 2.15
```

Comment:
this is the old constructor so we wont need the `@since` here but only for the api wise new stuff